corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -372,12 +372,29 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             self.parentView.setStickyQueryInputs();
         },
 
-        onRender: function () {
+        _initializeSelect2Dropdown: function () {
+            let placeHolderText;
+            switch (this.model.get('input')) {
+              case 'select1':
+                placeHolderText = 'Please select one';
+                break;
+              case 'select':
+                placeHolderText = 'Please select one or more';
```

Comment:
```suggestion
                placeHolderText = gettext('Please select one');
                break;
              case 'select':
                placeHolderText = gettext('Please select one or more');
```

Reply:
Thanks for catching it [a7d2320](https://github.com/dimagi/commcare-hq/pull/33423/commits/a7d23200788550c5c88f489b30b35f006eb64df9)